final_task/rss_reader/WorkWithCache.py

```diff
@@ -0,0 +1,72 @@
+import json
+import os
+
+from RssException import RssException
+# from Handler import News
+from Log import log_decore
+import urllib.request
+
+'''
+pass the dictionary to a function and write to a file json
+'''
+
+
+@log_decore
+def correct_title(title):
+    return title.replace('"', "_").replace("?", "_").replace(":", "_").replace("'", "_").replace(" ", "_")[:15]
```

Comment:
It is better to rethink this line, or at least leave a docstring here with example of what this function does and why it is needed